.github/workflows/auto_code_freeze_block_pr.yml

```
@@ -0,0 +1,42 @@
+name: Block PR on Code Freeze
+
+on:
+  pull_request:
+
+jobs:
+  check_for_code_freeze:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: octokit/request-action@v2.x
+      name: 'Get Milestones'
+      id: milestones
+      with:
+        route: GET /repos/{owner}/{repo}/milestones
+        owner: Datadog
```

not sure if it is case sensitive
```suggestion
        owner: DataDog
```